modules/ocf_postgres/manifests/init.pp

```diff
@@ -0,0 +1,23 @@
+class ocf_postgres {
+  class { 'postgresql::server':
+    postgres_password => hiera('postgres::root'),
+    #                      type    db       usr srcaddr   auth
```

Comment:
I'd probably put a link to the postgresql docs on this (https://www.postgresql.org/docs/current/static/auth-pg-hba-conf.html), since they are much more useful and give nice descriptions of each option and what they mean.